plugins/cilium-cni/cni-install.sh

```diff
@@ -110,7 +110,18 @@ find "${CNI_CONF_DIR}" -maxdepth 1 -type f \
 # the host.
 case "$CILIUM_CNI_CHAINING_MODE" in
 "aws-cni")
-  if [ $(find "${CNI_CONF_DIR}" -maxdepth 1 -type f -name '*.conf' -or -name '*.conflist' -or -name '*.cilium_bak' | wc -l) -eq 0 ]; then
+  tries=30
```

Comment:
any reason to have total time as 150 seconds ?

Reply:
I mean, the ideal answer is "never", but kubelet doesn't like it when postStart hooks never exit. I picked something at random; don't have a lot of evidence here either way :-(